.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: java
 
 script:
-  - ./gradlew test
\ No newline at end of file
+  - ./gradlew test
+
+os:
+  - osx
```

Comment:
what does this do?

Reply:
there's no desktop library for wpilib on linux so i have to tell it to run the build on osx